x-pack/legacy/plugins/ml/public/jobs/new_job_new/common/job_creator/single_metric_job_creator.ts

```diff
@@ -6,7 +6,7 @@
 
 import { SavedSearch } from 'src/legacy/core_plugins/kibana/public/discover/types';
 import { IndexPattern } from 'ui/index_patterns';
-import { parseInterval } from 'ui/utils/parse_interval';
+import { parseInterval } from '../../../../../common/util/parse_interval';
```

Comment:
You can try [path mapping](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping)

Reply:
I think a change like this would be outside the scope of this PR but is definitely something worth investigating to avoid these horrible relative paths.